pkg/server/plugin/keymanager/awskms/awskms.go

```diff
@@ -854,3 +852,20 @@ func makeFingerprint(pkixData []byte) string {
 	s := sha256.Sum256(pkixData)
 	return hex.EncodeToString(s[:])
 }
+
+// encodeKeyID maps "." and "+" characters to the asciihex value using "_" as
+// escape character. Currently KMS does not support those characters to be used
+// as alias name.
+func encodeKeyID(keyID string) string {
```

Comment:
I wonder if we should be more general purpose than this and do proper underscore escaping for any character outside of the allowed set for KMS aliases...

Reply:
Sounds good, I'll update it